### DISCUSSION TITLE
Provided key is corrupted or contains invalid data login the wallet

### DISCUSSION THREAD

**Discussion by trkriptocanavar** [IBAX-io/go-ibax#39] [Bug Submission]
![image](https://user-images.githubusercontent.com/80638208/149095180-d7d8d411-a19c-4496-9141-5247bbb0759b.png)

**Comment by HalloKyle** [comment]
> ![image](https://user-images.githubusercontent.com/80638208/149095180-d7d8d411-a19c-4496-9141-5247bbb0759b.png)

We cannot reproduce your issue. Please fill in the correct Key and try again，or just make a new wallet address instead.
